buildJenkinsfile

```diff
@@ -0,0 +1,86 @@
+@Library(value='iow-ecs-pipeline@2.2.0', changelog=false) _
+
+def p
+def deployConfigs
+
+pipeline {
+    agent {
+        node {
+            label 'team:iow'
+        }
+    }
+    parameters {
+        choice(choices: ['snapshots'], description: 'type of build', name: 'BUILD_TYPE')
```

Comment:
Good that this is only snapshots. We shouldn't allow releases. I think we could arguably leave out any of the release code below.

Reply:
Removed the stage where tags are pushed back up to the repo.